include/GoogleSync/GoogleSync.php

```diff
@@ -740,48 +780,92 @@ public function delEvent($event, $meeting_id)
         $statusCode = $return->getStatusCode();
 
         if ($statusCode >= 200 && $statusCode <= 299) {
-         // 2xx statusCode = success
-            $GLOBALS['log']->debug(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Received Success Status Code: ' . $statusCode . ' on delete.');
+            // 2xx statusCode = success
+            $this->logger->debug(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Received Success Status Code: ' . $statusCode . ' on delete.');
 
             // This removes the gsync_id reference from the table.
-            $sql = "UPDATE Meetings SET gsync_id = '' WHERE id = '" . $meeting_id . "'";
+            $sql = "UPDATE meetings SET gsync_id = '' WHERE id = {$valMeetingId}";
             $res = $this->db->query($sql);
+            if (!$res) {
+                $this->logger->fatal(__FILE__ . ':' . __LINE__ . ' ' . __METHOD__ . ' - ' . 'Failed to remove gsync_id from record' . $valMeetingId);
+            }
 
             $this->syncedList[] = $meeting_id;
             return true;
         } else {
```

Comment:
if you `return` from the function above one line of an `else`, the `else` beacame nonsence and unnecessary.
(Also, please use less `return`s /per methods so that the third party developers can debug your code more easy, it also makes less complexity of your code. - Not only here, everywhere..)